src/index.js

```diff
@@ -0,0 +1,39 @@
+// ----------------- Import Required Elements
+import taskList from './modules/taskarray.js';
+import * as element from './modules/elements.js';
+import * as taskFunction from './modules/functions.js';
+import './style.css';
+
+// ----------------- Collect data from Local Storage
+const store = localStorage.getItem('listOfTasks');
+
+if (store) {
+  taskList.push(...JSON.parse(store));
+} else {
+  taskList; // eslint-disable-line
```

Comment:
- [ ] OPTIONAL Please, try to remove any linter disables. Let the linter find the bugs in the early steps of your development 👍